### PR TITLE
Explicitly distinguish Callbacks from Events

### PR DESCRIPTION
**Pull request by philsbln**
Closes issue #775

### CODE REVIEW

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -1364,8 +1364,9 @@ Representation of Security Parameters in implementations should parallel that ch
 
 Security decisions, especially pertaining to trust, are not static. Once configured,
 parameters may also be supplied during connection establishment. These are best
-handled as client-provided callbacks. Security handshake callbacks that may be
-invoked during connection establishment include:
+handled as client-provided callbacks. 
+Callbacks block the progress of a connection, which distinguishes them from other Events in the transport system, while implementations may choose to realize them analogous.
```
Comment (britram) [inline draft-ietf-taps-interface.md:1368]:
```suggestion
Callbacks block the progress of a connection, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

**Review by tfpauly — APPROVED** [review]

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -1364,8 +1364,9 @@ Representation of Security Parameters in implementations should parallel that ch
 
 Security decisions, especially pertaining to trust, are not static. Once configured,
 parameters may also be supplied during connection establishment. These are best
-handled as client-provided callbacks. Security handshake callbacks that may be
-invoked during connection establishment include:
+handled as client-provided callbacks. 
+Callbacks block the progress of a connection, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md:1368]:
```suggestion
Callbacks block the progress of a Connection's establishment, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

**Review by mwelzl — CHANGES_REQUESTED** [review]
Minor suggestion

File: draft-ietf-taps-interface.md (hunk 2 above)
Comment (philsbln) [reply draft-ietf-taps-interface.md:1368]: I would prefer to leave the connection on lower case (as I am not sure that the connection object we refer to in upper case has/makes progress, but something in the underlaying transport system).

 ```suggestion
Callbacks block the progress of the connection establishment, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

Comment (mwelzl) [reply draft-ietf-taps-interface.md]: ok, this makes sense to me

**Review by mwelzl — APPROVED** [review]

**Review by britram — APPROVED** [review]

**Pull request merged**